Microsoft.AspNetCore.Components.Testing/TestHost.cs

```diff
@@ -23,6 +23,8 @@ public TestHost()
             });
         }
 
+        public IServiceProvider serviceProvider { get { return _serviceCollection.BuildServiceProvider(); } }
```

Comment:
For consistency with other APIs:

```suggestion
        public IServiceProvider Services { get { return _serviceCollection.BuildServiceProvider(); } }
```